Form-Controls/index.html

```diff
@@ -14,8 +14,48 @@ <h1>Product Pick</h1>
   </header>
   <main>
   <form>
-    <!-- write your html here-->
-    <!-- try writing out the requirements first-->
+    <header> <h1>Product Pick</h1>
+    </header>
+    <main>
+      <div class="wrapper">
+        <form id="product-form">
+        <div>
+        <label for="name">Name</label>
+        <input type="text" id="name" name="name" placeholder="Your name should be 9-20 characters long">
```

Comment:
Do the requirements say a name must be 9-20 characters long? 
https://www.kalzumeus.com/2010/06/17/falsehoods-programmers-believe-about-names/

Reply:
Alot of websites I visit usually have a name restriction and I just made an assumption following this statement from the read me file  "  What is the customer's name? I must collect this data, and validate it. But what is a valid name? I must decide something." 

I read the article and I will fix my mistake. Thank you for the article